utility_analysis_new/combiners.py

```diff
@@ -114,37 +118,96 @@ def __add__(self, other):
         return PartitionSelectionAccumulator(moments=moments_self +
                                              moments_other)
 
+    def compute_probability_to_keep(self, eps: float, delta: float,
+                                    max_partitions_contributed: int) -> float:
+        """Computes the probability that this partition is kept."""
```

Comment:
WDYT about adding a few more comments here like to describe that if probabilities aren't passed, there is an approximation?

Reply:
Thanks, it's a good point. Done

Reply:
🚀  Thank you!